lib/sms/sms.c

```diff
@@ -75,6 +79,15 @@ AT_MONITOR_ISR(sms_at_handler_cmt, "+CMT", sms_at_cmd_handler_cmt, PAUSED);
 AT_MONITOR_ISR(sms_at_handler_cds, "+CDS", sms_at_cmd_handler_cds, PAUSED);
 AT_MONITOR_ISR(sms_at_handler_cms, "+CMS", sms_at_cmd_handler_cms, PAUSED);
 
+void log_cms_error(int err) {
```

Comment:
I'd rename to start the function names with sms and comply with 
```suggestion
static void sms_cms_error_log(int err) {
```

Reply:
done